_artifacts/document-attributes.adoc

```diff
@@ -201,8 +201,8 @@ endif::OP[]
 :MANAGING_PROJECTS: Managing projects in {CENTRAL}
 :KIE_APIS: Interacting with {PRODUCT} using KIE APIs
 
-:DEPLOYING_OPENSHIFT_OPERATOR: Deploying a {PRODUCT} environment on {OPENSHIFT} using Operators
-:DEPLOYING_OPENSHIFT_TEMPLATES: Deploying a {PRODUCT} environment on {OPENSHIFT} using templates
+:DEPLOYING_OPENSHIFT_OPERATOR: Deploying a {PRODUCT} environment on {OPENSHIFT} version 4 using Operators
```

Comment:
Create and use a variable for the current OpenShift version.
remove 'version'. It should just be on OpenShift 4 using Operators.

@briandooley